### PR TITLE
feat: New design Auth/KYC pages

### LINKED ISSUE #1387: [Test Wallet - New Design]  - Auth/KYC pages
Make sure all Auth and KYC pages have the new look and feel of the design.

https://testnet-mockup.surge.sh/wallet/
https://testnet-mockup.surge.sh/wallet/styleguide/

### PR DESCRIPTION
**Pull request by Tymmmy**
### Context

- fixes #1387 

### Changes

- added some changes to KYC pages
- added changes to forgot password and verify emails
- added changes to 404 and not found pages

### CODE REVIEW

File: packages/wallet/frontend/src/components/layouts/AuthLayout.tsx [hunk 1]
```diff
@@ -26,24 +26,22 @@ const AuthLayout = ({ image, children }: AuthLayoutProps) => {
...
-          <div className="min-h-full flex-1 py-10">
-            <div className="mx-auto flex min-h-full w-full flex-col items-center sm:px-6 lg:px-20 xl:px-24">
-              {children}
-            </div>
+      <div className="flex min-h-full w-screen">
+        <div className="relative hidden w-0 flex-1 md:block">
+          <Image
+            fill
+            // sizes="(max-width: 768px) 100vw, (max-width: 1200px) 50vw, 33vw"
+            className="object-cover"
```
Comment (raducristianpopa) [inline packages/wallet/frontend/src/components/layouts/AuthLayout.tsx:34]: I feel like this is better on larger screens. Let me know what you think.
```suggestion
            className="md:object-cover lg:object-fill object-right"
```

File: packages/wallet/frontend/src/components/layouts/AuthLayout.tsx [hunk 2]
```diff
@@ -26,24 +26,22 @@ const AuthLayout = ({ image, children }: AuthLayoutProps) => {
...
+            src={imageSrc}
```
Comment (raducristianpopa) [inline packages/wallet/frontend/src/components/layouts/AuthLayout.tsx:35]: image

This image does not seem right also. I'm not seeing this in the Figma file.

File: packages/wallet/frontend/src/pages/kyc/personal.tsx
Comment (raducristianpopa) [inline packages/wallet/frontend/src/pages/kyc/personal.tsx:1]: Current display:
image

Mostly mobile:
- the theme toggle goes over the title
- we can remove some of the padding between the title and the logo - a lot of empty space
- some old colors are still being used (code wise - `text-pink`, `text-green-3`).
- for the notice `Denmark is ...`, lets center the text and make it a different color - right now its using the `text-pink` class which is not defined

File: packages/wallet/frontend/src/pages/kyc/proof.tsx
Comment (raducristianpopa) [inline packages/wallet/frontend/src/pages/kyc/proof.tsx:1]: Current display:
image


For this page:
- ditto above regarding the theme toggle and the title
- for the notice text, let's center it and add a small inline padding
- for the document selection (form), we can add a small inline padding as well - for me they are sticking to the sides.

File: packages/wallet/frontend/src/components/layouts/AuthLayout.tsx (hunk 2 above)
Comment (Tymmmy) [reply packages/wallet/frontend/src/components/layouts/AuthLayout.tsx:35]: My birdie looks like this:
image

**Pull request merged**